ModList/UI/Components/MarkdownText.cs

```diff
@@ -101,26 +101,31 @@ public UnityRenderer Renderer
 
         private (TMP_FontAsset font, string padding) LoadConfigFont(ModListConfig config)
         {
-            TMP_FontAsset asset = null;
+            TMP_FontAsset? asset = null;
             if (config.MonospaceFontPath != null)
             {
                 if (!File.Exists(config.MonospaceFontPath))
-                    Logger.md.Warn($"File '{config.MonospaceFontPath}' does not exist");
+                {
+                    // TODO: Inject logger for this
+                    // Logger.md.Warn($"File '{config.MonospaceFontPath}' does not exist");
+                }
                 else
                 {
-                    var ufont = new Font(config.MonospaceFontPath);
-                    asset = Helpers.TMPFontFromUnityFont(ufont);
+                    var uFont = new Font(config.MonospaceFontPath);
+                    asset = Helpers.Helpers.TMPFontFromUnityFont(uFont);
                 }
             }
+
             if (asset == null)
             {
-                if (!FontManager.TryGetTMPFontByFullName(config.MonospaceFontName, out asset, setupOsFallbacks: false)
-                 && !FontManager.TryGetTMPFontByFamily(config.MonospaceFontName, out asset, setupOsFallbacks: false))
+                if (!FontManager.TryGetTMPFontByFullName(config.MonospaceFontName, out asset, setupOsFallbacks: false) &&
+                    !FontManager.TryGetTMPFontByFamily(config.MonospaceFontName, out asset, setupOsFallbacks: false))
```

Comment:
I tend to prefer to have the operator at the beginning of the next line rather than the end of the last. Makes it easier to tell at a glance what the condition is.